src/sql/workbench/contrib/jobManagement/browser/notebooksView.component.ts

```diff
@@ -78,12 +78,12 @@ export class NotebooksViewComponent extends JobManagementView implements OnInit,
 
 	private _notebookCacheObject: NotebookCacheObject;
 	private rowDetail: RowDetailView<IItem>;
-	private filterPlugin: HeaderFilter<IItem>;
-	private dataView: Slick.Data.DataView<IItem>;
+	private filterPlugin: any;
```

Comment:
You should be able to keep these typed correctly - would prefer not to revert these.